atom/browser/web_contents_permission_helper.cc

```diff
@@ -82,29 +87,36 @@ void WebContentsPermissionHelper::RequestMediaAccessPermission(
   auto callback = base::Bind(&MediaAccessAllowed, request, response_callback);
   // The permission type doesn't matter here, AUDIO_CAPTURE/VIDEO_CAPTURE
   // are presented as same type in content_converter.h.
-  RequestPermission(content::PermissionType::AUDIO_CAPTURE, callback,
+  auto rfh = content::RenderFrameHost::FromID(request.render_process_id,
+      request.render_frame_id);
+  RequestPermission(content::PermissionType::AUDIO_CAPTURE, callback, rfh,
```

Comment:
I think we should check for a null rfh here and make the callback with access denied if it is

Reply:
actually after looking at the chrome code I think we should just default to the main frame here